SU-SWS/saa_victoria_subtheme
# ADAPT-683: Fixed cut off fb icon when tabbing.

# READY FOR REVIEW

# Summary
- Fix outline of footer fb icon when tabbing

# Review By (Date)
- 9/16

# Urgency
- low

# Steps to Test

1. Pull down the branch
2. Clear the site cache
3. tab through the footer column one and make sure that the outline around the facebook icon is not cut off on the left side.

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket ADAPT-683
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
